src/Pages/AboutUs.js

```diff
@@ -1,12 +1,22 @@
 import React, { useState } from "react";
 
 const AboutUs = () => {
-    
-    return (
-        <div className="aboutUsContainer">
-            <h1> "Meet our dynamic team of four students who embarked on an exciting journey through a rigorous coding bootcamp. Comprising Derek, Jacob, Markayel, and Corey, our team is fueled by a relentless passion for learning and a shared commitment to mastering the art of programming. Motivated by a common goal to excel in the world of coding, we dive into each lesson with enthusiasm, embracing challenges as opportunities to grow. With a collective dedication to honing our programming skills, we're not just students; we're a tight-knit community of aspiring developers eager to make a mark in the ever-evolving tech landscape." </h1>
-        </div>
-    )
-}
+  return (
+    <div className="aboutUsContainer">
+      <h1>
+        {" "}
```

Comment:
A little curious why you have an empty string here?

Reply:
When Prettier ran on this file there was a space between the <h1> tag and the text on both ends and Prettier turned that space into {" "}. How should i approach making changes to material that isn't listed on a Trello card?

Reply:
Since this is already merged, just create a new branch and instead of providing a Trello card, just let me know that it was a small fix.  Or you can make a new Trello card for it as well.